cache/ttlmemory_test.go

```diff
@@ -0,0 +1,60 @@
+package cache_test
```

Comment:
nit: it's more conventional to not have separate packages for the test and just reuse the package name directly, unless there's a particular reason to separate them.

Reply:
I wanted it to test it like it would be used in Evergreen. I can combine them